### DISCUSSION TITLE
Intent 10 analysis

### DISCUSSION THREAD

**Discussion by Kevsy** [camaraproject/EdgeCloud#181] [General]
### Developer intents: Provisioning intents

| Intent 10 | 5SED |5MEE |EXRA|EXRC| EXCA | EXSM|  GMEC |
| -------- |------| ----|----| ---|------|-----|-------|
| 6. "I can ask the operator to link artifacts to the applications when onboarding my applications" GMEC."  | No           |No    | No          | ?     | ?            | ?     |Yes   |

| GMEC        | Details       |Comments|
| --------------|-------------| -----------|
| (to complete) | (to complete) | (to complete) |

(others add as appropriate)

**Comment by gunjald** [comment]
The application management API provide the app provider the capability to reuse the artefacts across applications by linking artefacts to the applications.
